99_post_install.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -xe
+
+source logging.sh
+source common.sh
+
+KUBECONFIG=${KUBECONFIG:-ocp/auth/kubeconfig}
+
+for node_type in master node; do
+  # Disable auto reboot hosts in order to apply several mcos at the same time
+  oc patch --type=merge --patch='{"spec":{"paused":true}}' machineconfigpool/${node_type}
+
+  # Apply mcos
+  oc create -f assets/post-install/*-${node_type}.yaml
+
+  # Enable auto reboot
+  oc patch --type=merge --patch='{"spec":{"paused":false}}' machineconfigpool/${node_type}
+
+  # This sleep is required because the machine-config changes are not immediate
+  sleep 30
```

Comment:
What needs to happen in these 30 seconds?  There's a wait right after this.  Will that one fail if this wait doesn't happen?  What does it fix?

Reply:
If the `oc wait` is performed just after applying the machine configs, it finishes as the machine config operator takes a while to notice the changes and create the rendered machine config. 30 seconds is just random, but definitely in my tests I needed to wait a few seconds.